rpc/src/views.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_types::account_config::AccountResource;
+use libra_types::byte_array::ByteArray;
+use serde::Serialize;
+
+#[derive(Serialize)]
+pub struct AccountView {
+    balance: u64,
+    sequence_number: u64,
+    authentication_key: ByteArray,
```

Comment:
What does ByteArray serialize into?  JSON doesn't have support for native bytes, we either have to send Base64 encoded string, or hex encoded string, I think we should be more clear on what end result is here.

Reply:
it actually returns array of integers now :) 
hex encoded string makes sense here though